providers/lambdalabs/node.go

```diff
@@ -288,12 +288,6 @@ func (n *NodeRuntime) InitNode(ctx context.Context, sshKey []types.SSHKey, nodeT
 		return types.Node{}, fmt.Errorf("failed to launch instance")
```

Comment:
Maybe you should create some global errors that could be handled somewhere above?
like 
```go
var ErrInstanceLaunchFailed = errors.New("failed to launch instance")
```

and then you could wrap it in this return as 
```go
return types.Node{}, fmt.Errorf("%w; %s", ErrInstanceLaunchFailed, "sshKey is required")
```

Personally, I think that this approach has 2 big PROS:
- You can handle those errors in a special way in the code somewhere above
- You are adding more context to the error, which is helpful while debugging.

As the side note, I think that golang in the next versions is going to have errors with a stack trace (it came up from developers survey that errors handling need some changes).